Cargo.toml

```diff
@@ -306,6 +308,10 @@ debug-assertions = false
 overflow-checks = false
 rpath = false
 
+[profile.release.package.server]
```

Comment:
How about only setting `codegen-units = 1` to packages `tikv-server` and `tikv-ctl`?

Reply:
I don't know what it could affect. Instead of `server` which is not in the critical path, if we increase the codegen-units of other component crates, will they become less optimized and downgrade the whole performance?

Reply:
I'm not sure either. I wonder whether `lto = true + codegen-units =1` in the final package can produce good enough binaries. But I'm OK leave it for future investigation.